drivers/spi/spi_ll_stm32.c

```diff
@@ -454,6 +454,19 @@ static void spi_stm32_cs_control(const struct device *dev, bool on)
 #endif
 }
 
+#ifdef CONFIG_SPI_STM32_INTERRUPT
+static void spi_stm32_disable_irqs(SPI_TypeDef *spi)
+{
+	ll_func_disable_int_tx_empty(spi);
+	ll_func_disable_int_rx_not_empty(spi);
+	ll_func_disable_int_errors(spi);
+
+#if DT_HAS_COMPAT_STATUS_OKAY(st_stm32h7_spi)
+	LL_SPI_DisableIT_EOT(spi);
```

Comment:
Following functions are now missing. This is not justified in the commit message.
	LL_SPI_DisableIT_DXP(spi);	
	LL_SPI_DisableIT_TXTF(spi);

Reply:
@GeorgeCGV requested to remove them because they are not enabled before. You commented as well that you agreed in that. 

In https://github.com/zephyrproject-rtos/zephyr/pull/66503#discussion_r1426617281:

> DXP and TXTF interrupts are never enabled. [...]